src/binary/utils/logger-with-error-boundary.ts

```diff
@@ -12,6 +12,14 @@ export function loggerWithErrorBoundary<T>(
     try {
       await command(flags, logger);
     } catch (error) {
+      /*
```

Comment:
Nice to add this. If you include the argument name when creating InvalidOptionError, then maybe it makes sense to do some special formatting here. If you create a nice message, I don't see the need

Reply:
I'll leave the comment for now and remove it in a future iteration.